src/layout/header/Header.tsx

```diff
@@ -154,15 +175,25 @@ export const Header = () => {
                   {t('basic_data.basic_data')}
                 </MenuButton>
               )}
-              {(hasCuratorRole(user) || user?.isNviCurator) && (
-                <MenuButton
-                  color="inherit"
-                  data-testid={dataTestId.header.tasksLink}
-                  isSelected={currentPath.startsWith(UrlPathTemplate.Tasks)}
-                  to={UrlPathTemplate.Tasks}
-                  startIcon={<AssignmentIcon />}>
-                  {t('common.tasks')}
-                </MenuButton>
+              {(isTicketCurator || user?.isNviCurator) && (
+                <Badge
+                  badgeContent={notificationCounts || null}
+                  color="info"
+                  sx={{
+                    '& .MuiBadge-badge': {
+                      right: 20,
+                      top: 20,
+                    },
+                  }}>
+                  <MenuButton
+                    color="inherit"
+                    data-testid={dataTestId.header.tasksLink}
+                    isSelected={currentPath.startsWith(UrlPathTemplate.Tasks)}
+                    to={UrlPathTemplate.Tasks}
+                    startIcon={<AssignmentIcon />}>
+                    {t('common.tasks')}
+                  </MenuButton>
+                </Badge>
```

Comment:
Funker dette like greit? Blir ikke helt samme position, men ganske close.
```suggestion
                <MenuButton
                  color="inherit"
                  data-testid={dataTestId.header.tasksLink}
                  isSelected={currentPath.startsWith(UrlPathTemplate.Tasks)}
                  to={UrlPathTemplate.Tasks}
                  startIcon={
                    <Badge badgeContent={notificationCounts || null} color="info">
                      <AssignmentIcon />
                    </Badge>
                  }>
                  {t('common.tasks')}
                </MenuButton>
```

Reply:
Ser nå at den bumper hele knappen litt ned, så det er jo ikke helt heldig :/

Reply:
Mye finere kode da! 🙌 Selv om det er et lite mysterie hvorfor `startIcon` kommer akkurat der 😅 Ser også at hoved-ikonet hopper en piksel-ish ja, som er litt synd :/

Reply:
ah, nå først skjønner jeg hva som skjer ja, den wrappes rundt selve ikonet såklart. Syns det er en mer logisk løsning enn det jeg hadde først, men så var det det lille hoppet da... 🤔